src/constraints.js

```diff
@@ -0,0 +1,135 @@
+import { _ } from 'underscore'
```

Comment:
@zhenyab I know the problem :). Please remove underscore and use lodash. Here is an example of lodash use:

https://github.com/okfn/data-quality-dashboard/blob/master/app/config.js#L3

Again, I know this was old code, but we have standardized on lodash almost everywhere, and I'd like to get that sorted here for this.